### ISSUE TITLE
far2l/src/plug/plugins.cpp fails to compile when wchar_t is unsigned

### ISSUE THREAD

**Issue by danfe** [elfmz/far2l#1302]
The `wchar_t` is implementation-defined type, and could be *unsigned* on some architectures.  In this case, the following code around line 1612 inside `far2l/src/plug/plugins.cpp` breaks:
```c++
    const auto &Setting = kfh.GetString(SettingsSection, SettingName);
    WCHAR Letter[2] = {Setting.empty() ? 0 : Setting[0], 0};
```
Could it be because the compiler uses incorrect `GetString()` method to infer the type for `&Setting`?  While both standard and wide versions of it exist, the `std::wstring` prototype in `utils/include/KeyFileHelper.h` is missing the `const wchar_t *def = L""` part, so `std::string` version gets picked which does not match wide `Letter`.

**Comment by shmuz** [comment]
@danfe , are you looking at the latest source code?
I guess the line you mentioned  is not near line 1612 and now it is here: https://github.com/elfmz/far2l/blob/040c03d9a32241c86abce04df7dfe29f7e848179/far2l/src/plug/plugins.cpp#L1625

**Comment by danfe** [comment]
> are you looking at the latest source code?

No, I was not, mea culpa.  Apparently this had been fixed in f49e247, sorry for the noise.
